source/topics/troubleshooting/support-ticket.md

```diff
@@ -0,0 +1,11 @@
+When submitting a support request to Aptible, keep the following best practices in mind:
+
+1) If you provide us with as much detail as possible we'll be able to troubleshoot your issue much faster.  When appropriate, please provide  
+
+- Relevant handles (app, account, database etc) 
```

Comment:
These bullets should be indented one or two spaces, to create a nested unordered list.